configure

```diff
@@ -893,6 +905,16 @@ def configure_mips(o):
   o['variables']['mips_fpu_mode'] = options.mips_fpu_mode
 
 
+def check_gcc_version(version_checked):
```

Comment:
Maybe `gcc_version_ge`? Though I'm fine with this too. (ge for greater-or-equals in python)

Reply:
Agree and tried to resolve it.